vendor-bin/psalm/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "require-dev": {
```

Comment:
Thanks for adding these as vendor-bin deps, makes eveything much cleaner! :blue_heart: